ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -276,6 +276,7 @@
 "CARDIO_OUTDOOR_INSTRUCTION_TITLE" = "Please move outdoors";
 "CARDIO_OUTDOOR_INSTRUCTION_TEXT" = "Your phone GPS indicates that you are currently indoors. Please step outdoors to before continuing.";
 "CARDIO_OUTDOOR_ALREADY_OUTSIDE" = "I am already outdoors.";
+"CARDIO_OUTDOOR_STAY_INDOORS" = "I want to stay inside.";
```

Comment:
The instructions specifically request that the user walk outdoors because it allows for more accurate location measurement. 

However, this step is now just "You appear to be indoors. Please let us know if (a) I have now moved outside (b) I am already outside and your sensors are crap or (c) I want to stay indoors." and then it continues, no matter which answer you give.